tests/10apidoc/01register.pl

```diff
@@ -61,6 +61,30 @@
       });
    };
 
+test "POST /createUser can get or create a user",
+   requires => [ $main::API_CLIENTS[0] ],
+
+   do => sub {
+      my ( $http ) = @_;
+
+      $http->do_request_json(
+         method => "POST",
+         uri    => "/unstable/createUser",
+
+         content => {
+            localpart     => "01register-user"
+            displayname     => "01register_user x",
+            duration_seconds => 200,
```

Comment:
Sure I will fix this and also I should say I have never used perl before. Sorry about my very clumsy mistakes here.